src/fuzz/simple_fuzz_test.cc

```diff
@@ -0,0 +1,137 @@
+  if (setenv("TZDIR", tzdataDir.c_str(), OVERWRITE)) {
+    return false;
+  }
+  return true;
+}
+#endif
+
+const std::string server_address = "localhost:1234";
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  #ifdef __OSS_FUZZ__
+    static bool Initialized = DoOssFuzzInit();
+    if (!Initialized) { std::abort(); }
+  #endif
+
+  Server::Options options;
+  options.server_address = server_address;
+  std::unique_ptr<Server> server = Server::Create(options);
+  if (!server) {
+    return EXIT_FAILURE;
+  }
+
+  std::cout << "Server Up, Executing Test Query" << std::endl;
+
+  try {
+    // This is the connection to the emulator.
+    ConnectionOptions emulator_connection;
+    emulator_connection.set_endpoint(server_address)
+        .set_credentials(grpc::InsecureChannelCredentials());
+
+    // First we create an instance to create our database on.
+    google::cloud::spanner::Instance instance("emulator", "emulator");
+    google::cloud::spanner::InstanceAdminClient instance_client(
+        google::cloud::spanner::MakeInstanceAdminConnection(
+            emulator_connection));
+
+    auto instance_or = instance_client.CreateInstance(
+            google::cloud::spanner::CreateInstanceRequestBuilder(instance,
+                                                                 "emulator")
+                .SetDisplayName("emulator")
+                .SetNodeCount(1)
+                .SetLabels({{"label-key", "label-value"}})
+                .Build())
+                .get();
+    if (!instance_or) {
+      throw std::runtime_error(instance_or.status().message());
+    }
+    std::cout << "Created instance [" << instance << "]\n";
+
+    // Then we create a simple database.
+    google::cloud::spanner::Database database(instance, "test-db");
+
+    auto admin_client =
+        DatabaseAdminClient(google::cloud::spanner::MakeDatabaseAdminConnection(
+            emulator_connection));
+    auto db_or =
+        admin_client.CreateDatabase(database, {R"sdl(
+          CREATE TABLE Singers (
+              SingerId   INT64 NOT NULL,
+              FirstName  STRING(1024),
+              LastName   STRING(1024),
+              SingerInfo BYTES(MAX)
+          ) PRIMARY KEY (SingerId))sdl",
+                                               R"sdl(
+          CREATE TABLE Albums (
+              SingerId     INT64 NOT NULL,
+              AlbumId      INT64 NOT NULL,
+              AlbumTitle   STRING(MAX)
+          ) PRIMARY KEY (SingerId, AlbumId),
+              INTERLEAVE IN PARENT Singers ON DELETE CASCADE)sdl"})
+            .get();
+    if (!db_or) throw std::runtime_error(db.status().message());
+    std::cout << "Created database [" << database << "]\n";
+
+    google::cloud::spanner::Client client(
+        google::cloud::spanner::MakeConnection(database, emulator_connection));
+
+    std::string query = absl::Substitute("INSERT INTO Singers (FirstName) VALUES ($0)", std::string((char*)Data, Size));
```

Comment:
Google style guide limits lines to 80 characters.

Reply:
It may be worth just running "clang-tidy --format-style=google --fix" on the file.  You may need to install clang-tidy first.  This will apply this fix and other automatic Google-style related fixes.